streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -307,97 +308,6 @@ public void testPauseResume() {
         assertEquals(0, consumer.paused().size());
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testMaybePunctuateStreamTime() {
```

Comment:
Why is this removed but not updated? Is is redundant?

Reply:
It is renamed to another test function :)